transcom/mymove
# set up key press handling so that backspace would clear out the field

## [Agility ticket](tbd)

## Summary

I found the common component that is used for all location-related drop-down selection fields and added functionality that will allow the user to hit the BACKSPACE key on their keyboard to retype the search query as the previous functionality was not clear to the user how to retype the search query.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Automated Testing:
Added test criteria to existing Onboarding and Orders tests.

Manual Testing:
1. Access the MilMove app
2. Create a new user.
3. As you are following the "new user setup" process, you should notice the modified selection box for selecting the location/address.
4. In addition, you may want to check some other areas where the location/address selection drop-down is.
a. Closeout Office Input
b. Duty Location
c. Orders Panel
d. Edit Profile

### Frontend


### Backend


## Screenshots

![Screenshot 2023-10-27 at 8 00 09 AM](https://github.com/transcom/mymove/assets/147739091/534cb880-2c6d-4689-9ebe-bdae71ec1a10)
